.github/workflows/lint.yml

```diff
@@ -0,0 +1,31 @@
+name: lint
+on:
+    pull_request:
```

Comment:
hmm this isn't running on the PR yet. Maybe it needs to merge before it activates